feature/experimental/tunnel/otg_tests/tunnel_interface_based_disable_enable_test/tunnel_interface_disable_enable_test.go

```diff
@@ -0,0 +1,1558 @@
+		keys := []string{}
+		for m := range dut1FinalFitStats {
+			keys = append(keys, m)
+		}
+		sort.Strings(keys)
+		for j, key := range keys {
+			VerifyLoadbalance(t, 4, p.trafficRate, p.trafficDuration, ftiIntfCount, int64(dut1InitialFitStats[key]), int64(dut1FinalFitStats[key]), loadbalance[j])
+		}
+
+	}
+
+	if deviations.TunnelStatePathUnsupported(dut2) {
+		dut2FinalFitStats := map[string]uint64{}
+		dut2FinalFitStats["dut2Fti0InPkts"] = gnmi.Get(t, dut2, gnmi.OC().Interface("fti0").Subinterface(0).Counters().InPkts().State())
+		dut2FinalFitStats["dut2Fti1InPkts"] = gnmi.Get(t, dut2, gnmi.OC().Interface("fti1").Subinterface(0).Counters().InPkts().State())
+		dut2FinalFitStats["dut2Fti2InPkts"] = gnmi.Get(t, dut2, gnmi.OC().Interface("fti2").Subinterface(0).Counters().InPkts().State())
+		dut2FinalFitStats["dut2Fti3InPkts"] = gnmi.Get(t, dut2, gnmi.OC().Interface("fti3").Subinterface(0).Counters().InPkts().State())
+		dut2FinalFitStats["dut2Fti4InPkts"] = gnmi.Get(t, dut2, gnmi.OC().Interface("fti4").Subinterface(0).Counters().InPkts().State())
+		dut2FinalFitStats["dut2Fti5InPkts"] = gnmi.Get(t, dut2, gnmi.OC().Interface("fti5").Subinterface(0).Counters().InPkts().State())
+		dut2FinalFitStats["dut2Fti6InPkts"] = gnmi.Get(t, dut2, gnmi.OC().Interface("fti6").Subinterface(0).Counters().InPkts().State())
+		dut2FinalFitStats["dut2Fti7InPkts"] = gnmi.Get(t, dut2, gnmi.OC().Interface("fti7").Subinterface(0).Counters().InPkts().State())
+
+		t.Logf("Decapsulating router Final fti0 interface InPkts stats: %d\n", dut2FinalFitStats["dut2Fti0InPkts"])
+		t.Logf("Decapsulating router Final fti1 interface InPkts stats: %d\n", dut2FinalFitStats["dut2Fti1InPkts"])
+		t.Logf("Decapsulating router Final fti2 interface InPkts stats: %d\n", dut2FinalFitStats["dut2Fti2InPkts"])
+		t.Logf("Decapsulating router Final fti3 interface InPkts stats: %d\n", dut2FinalFitStats["dut2Fti3InPkts"])
+		t.Logf("Decapsulating router Final fti4 interface InPkts stats: %d\n", dut2FinalFitStats["dut2Fti4InPkts"])
+		t.Logf("Decapsulating router Final fti5 interface InPkts stats: %d\n", dut2FinalFitStats["dut2Fti5InPkts"])
+		t.Logf("Decapsulating router Final fti6 interface InPkts stats: %d\n", dut2FinalFitStats["dut2Fti6InPkts"])
+		t.Logf("Decapsulating router Final fti7 interface InPkts stats: %d\n", dut2FinalFitStats["dut2Fti7InPkts"])
+
+		t.Logf("Verify Overlay loadbalancing at decapsulation router- Verify the tunnel interfaces traffic/flow for equal distribution for optimal load balancing ) ")
+		keys := []string{}
+		for n := range dut2FinalFitStats {
+			keys = append(keys, n)
+		}
+		sort.Strings(keys)
+		for k, key := range keys {
+			VerifyLoadbalance(t, 4, p.trafficRate, p.trafficDuration, ftiIntfCount, int64(dut2InitialFitStats[key]), int64(dut2FinalFitStats[key]), loadbalance[k])
+		}
+	}
+}
+
+func disableAndEnableTunnelInterface(t *testing.T, dut *ondatra.DUTDevice, state bool) {
+
+	dutIntfs := []struct {
+		desc     string
```

Comment:
the desc value is the same for all interfaces, so just inline the string "tunnel_endpoint" and simplify this to a slice of strings